vulcanai2/models/basenetwork.py

```diff
@@ -291,31 +290,26 @@ def _init_optimizer(self, optim_spec):
     def _init_criterion(criterion_spec):
         return criterion_spec()
 
-    def _init_trainer(self):
+    def _init_trainer(self, train_loader):
         self.optim = self._init_optimizer(self._optim_spec)
         self.criterion = self._init_criterion(self._criter_spec)
-        self.epoch = 0
+        self.valid_interv = 2 * len(train_loader)
 
-    def fit(self, train_loader, val_loader, epochs, 
+    def fit(self, train_loader, val_loader, epochs,
             retain_graph=None, valid_interv=4, plot=False):
         """
         Trains the network on the provided data.
         :param train_loader: The DataLoader object containing the training data
         :param val_loader: The DataLoader object containing the validation data
         :param epochs: The number of epochs
         :param retain_graph: Specifies whether retain_graph will be true when .backwards is called.
-        :param valid_interv:
+        :param valid_interv: Specifies when validation should occur. Not yet implemented.
```

Comment:
I believe @priyatharsan implemented it recently. We just need to fix how we omit printing validation scores that don't exist in-between the interval